android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -1211,15 +1211,18 @@ public void onPlayerError(ExoPlaybackException e) {
                             decoderInitializationException.codecInfo.name);
                 }
             }
+            Log.d("SkyTVPlayer", "TYPE_RENDERER");
         } else if (e.type == ExoPlaybackException.TYPE_SOURCE) {
             errorString = getResources().getString(R.string.unrecognized_media_format);
+            Log.d("SkyTVPlayer", "TYPE_SOURCE");
         }
         eventEmitter.error(errorString, ex);
         playerNeedsSource = true;
         playerNeedsNewLicence = true;
-        if (isBehindLiveWindow(e)) {
+        if (isBehindLiveWindow(e) || e.type == ExoPlaybackException.TYPE_RENDERER) {
             clearResumePosition();
             initializePlayer();
```

Comment:
we dont need to releasePlayer?

Reply:
ah sorry so its out of the box code fix. all good

Reply:
It was enough to restart the stream only